Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/WriterTests/JsonWriterTests.cs

```diff
@@ -32,77 +45,105 @@ public void ExistsTests()
         }
 
         [Fact]
-        public void SetTokenTests()
+        public void SetToken()
         {
             // ARRANGE
-            var rootNode = new JObject();
-            var jsonWriter = new JsonWriter(rootNode);
+            ITemplateWriter jsonWriter = new JsonWriter();
 
             // ACT
-            jsonWriter.SetToken("Person.Name.FirstName", new JValue("ABC"));
-            jsonWriter.SetToken("Person.Name.LastName", new JValue("XYZ"));
-            jsonWriter.SetToken("Person.Age", new JValue(50));
-            jsonWriter.SetToken("Person.DOB", new JValue(new DateTime(2000, 1, 1)));
-            jsonWriter.SetToken("Person.PhoneNumbers", new JArray(new List<string> { "123", "456", "789" }));
+            jsonWriter.SetToken("Person.Name.FirstName", "ABC", TokenType.Other);
```

Comment:
minor/non-blocking - don't think we need the `TokenType.Others` here since we added a default due to PR fedback

Reply:
Done